index.test.js

```diff
@@ -14,19 +16,15 @@ describe('Env Middleware', () => {
         }),
       );
 
-    handler({}, {}, (err) => {
-      expect(err).toEqual(new ReferenceError('Environment variable TEST1 is missing'));
-      done();
-    });
+    await expect(handler({}, {})).rejects.toEqual(new ReferenceError('Environment variable TEST1 is missing'));
   });
 
-  test('it passes the enviroment variable values into context by default', (done) => {
+  test('it passes the enviroment variable values into context by default', async () => {
     process.env.TEST2 = 'testvalue';
 
     const handler = middy(
       (_, context) => {
         expect(context).toHaveProperty('test', 'testvalue');
```

Comment:
Do you need to `expect.assertions(1);` here, to ensure the callback is called and calls your assertion?